lib/db.js

```diff
@@ -190,7 +190,6 @@ function net_db(params, callback) {
       logger.warning('Problem contacting DITCH server: ' + err.message + "\n" + util.inspect(err.stack));
       return callback(err);
     });
-    req.end();
```

Comment:
@trepel Right, looks like it was introduced with this commit: https://github.com/fheng/fh-mbaas-api/commit/3170fe119fa7abc981fe3d10c7e05afb9d80df42#diff-be228df81b99a384aa50ad50bde7fca0R114

No real explanation given though.